README.md

```diff
@@ -145,3 +145,42 @@ A server **MAY** respond with other HTTP status codes.
 A server **MAY** include error details with error responses.
 
 A server **MUST** prepare responses, and a client **MUST** interpret responses, in accordance with HTTP semantics.
+
+### Example response document
+
+This example shows a response document from the [example request document](#example-request-document). Both the post and the tag which was requested to be created are included as primary data. The additional already existing tag shouldn't be placed in the included data since the response only should return created resources, but it could be included if requested via the include query parameter.
```

Comment:
This wasn't fully precise yet. Unless the `include` query parameter is set in the request, it's up to the server if the existing tag is included in the response document under `included` key or not. I tried rewording it.

```suggestion
This example shows a response document from the [example request document](#example-request-document). Both the post and the tag which were requested to be created are included in the response document as primary data. The existing tag is not included in the response. A server could include it under `included` key unless the client requested inclusion of specific related resources explicitly using the `include` query parameter.
```